### DISCUSSION TITLE
Adding tooltip with customization capabilities

### DISCUSSION THREAD

**Discussion by pyeh-seg** [grafana/grafana#64263] [Dashboards, Panels & Visualizations]
Any chance we could have tooltip back? Also, I hope we could have the ability to customize tooltip based on query result.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is fairly vague - where is a tooltip now missing? Feel free to make an [issue](https://github.com/grafana/grafana/issues) with more details to request the feature.
